tests/aws/services/cloudformation/api/test_update_stack.py

```diff
@@ -407,3 +408,43 @@ def test_update_with_rollback_configuration(deploy_cfn_template, aws_client):
 
     # cleanup
     aws_client.cloudwatch.delete_alarms(AlarmNames=["HighResourceUsage"])
+
+
+@markers.aws.validated
+def test_diff_after_update(deploy_cfn_template, aws_client, snapshot):
+    template_1 = textwrap.dedent("""
+    Resources:
+        SimpleParam:
+            Type: AWS::SSM::Parameter
+            Properties:
+                Value: test
+                Type: String
+    """)
+    template_2 = textwrap.dedent("""
+    Resources:
+        SimpleParam1:
+            Type: AWS::SSM::Parameter
+            Properties:
+                Value: test1
+                Type: String
```

Comment:
I found the change from `test` to `test1` a little easy to miss on first read. Perhaps make the two `Value` values more significantly different, e.g. `before-stack-update` and `after-stack-update`?